scylla-cql/src/frame/protocol_features.rs

```diff
@@ -1,22 +1,53 @@
+//! Implementation and negotiation of extensions to the CQL protocol.
+
 use std::borrow::Cow;
 use std::collections::HashMap;
 
 const RATE_LIMIT_ERROR_EXTENSION: &str = "SCYLLA_RATE_LIMIT_ERROR";
+/// The extension used to add metadata for LWT optimization.
 pub const SCYLLA_LWT_ADD_METADATA_MARK_EXTENSION: &str = "SCYLLA_LWT_ADD_METADATA_MARK";
```

Comment:
You could describe what this metadata contains, or link to docs of this extension in scylla repo.